.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: false # Explicitly use container-based infrastructure
+
+language: go
+
+go:
+  - 1.6.2
+
+env:
+    - BUILDER_VERSION=0.11.0
+
+cache:
+    directories:
+        - ./builder-cache
+        
+notifications:
+    email: false
+
+before_install:
+    # Grab builder from the cache, or download it if we can't.
+    - cp ./builder-cache/builder-$BUILDER_VERSION ./builder || wget https://downloads.giantswarm.io/builder/$BUILDER_VERSION/builder && chmod +x ./builder
+    - chmod +x ./builder
+    - export PATH=$PATH:$PWD
```

Comment:
What about having this added to the Makefile? Then the CI target can depend on such setup targets.